baseline/TIPS_AND_TRICKS.md

```diff
@@ -0,0 +1,33 @@
+
+Some of the factors include:
+
+- Some checks in the audit are a direct report of a certain result, most notably with WARNING/medium.  If the item in the audit is a report WARNING, it can not change in the baseline audit.  If the audit is a Tenable published audit, these items can generally be identified by having a "NOTE:" in the description that the check was not run. The code inside the audit will have the opening tag being similar to `<report type:"WARNING">`.
+- Audits can contain conditional logic that will provide results based on a setting on the target being scanned.  When rescanning the "gold image" system with a baseline audit, the same conditional logic should work, and the results should become a PASSED/info.  But when scanning other hosts, they may take a different conditional path that can provided results that were not present in the original "gold image" results.
+- If the output of an audit check has dynamic data in it, such as timestamps, an known good value of a baseline will not work.  This is due to the output with a time stamp will change with every execution of the scan, and thus would not match the baseline run.
+- When creating a baseline audit, the original range or regular expression is now abandoned and a absolute value is now used.  This could be the benchmark accepted a password lenght of anything over 8 characters, but the "gold image" had 7 characters set.  The baseline audit will fail on anything that is not 7 characters.  This can be adjusted by adding more known good values, but in the end, it will be looking for absolute values.
+
+
+## Getting the Source Audit
+
+The use of this script requires that there is access to the original audit file used during the scanning of the "gold image".  This is required in order to get the best baseline audit file created based on the results of that scan.
+
+If using a custom audit, having access to the source is not an issue.
+
+If wanting to use a Tenable, or other third party audit, the original method was to get the source audit from a download site, customize it with the organizations policy values, and treat it as a custom audit when scanning.  The requirement to verify the custom policy values adds overhead and possibilties for missing values, or mis-typing values.
+
+As of Marge of 2019, a debug opotion was enable that allows the source audit to be presented in the Debugging Log Report.  With this, you can run the "gold image" scan using any audit file and enabling Debug Plugin option in the policy.  When the results are done, the source audit can be saved out of the Debugging Log Report.  The only modification that must be done is to remove the first line in the file as it contains a timestamp.
+
+
+## Custom Audit Content
+
+When creating a custom audit for use as a baseline, the following tips will help get quality results:
+
+* Make the audit relatively flat and uses few to no conditionals.  Conditionals may change the results presented on different targets.
```

Comment:
Update grammar to:

Make the audit relatively flat and use few to no conditionals.